File: verification-packs/polyglot-30-route-formal-equivalence-v1/evidence/routes/python-to-go/certification/artifacts/development/source-runtime/source_harness.py
import json
import pricing
actual_0 = pricing.calculate(100, 20)
assert actual_0 == 120
print("ELMOS_OBSERVATION\tjson\t" + json.dumps({"case_id": 0, "value": actual_0}, sort_keys=True, separators=(",", ":")))
actual_1 = pricing.calculate(-1, 5)
assert actual_1 == 0
print("ELMOS_OBSERVATION\tjson\t" + json.dumps({"case_id": 1, "value": actual_1}, sort_keys=True, separators=(",", ":")))
actual_2 = pricing.calculate(7, -2)
assert actual_2 == 5
print("ELMOS_OBSERVATION\tjson\t" + json.dumps({"case_id": 2, "value": actual_2}, sort_keys=True, separators=(",", ":")))

File: routes/rust-to-csharp/certification/artifacts/development/source-runtime/source_harness.rs
include!("pricing.rs");

fn main() {
    let actual_0 = calculate(100, 20);
    assert!(actual_0 == 120, "case 0");
    println!("ELMOS_OBSERVATION\t0\trust-debug\t{:?}", actual_0);
    let actual_1 = calculate(-1, 5);
    assert!(actual_1 == 0, "case 1");
    println!("ELMOS_OBSERVATION\t1\trust-debug\t{:?}", actual_1);
    let actual_2 = calculate(7, -2);
    assert!(actual_2 == 5, "case 2");
    println!("ELMOS_OBSERVATION\t2\trust-debug\t{:?}", actual_2);
}
